packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -321,6 +304,10 @@ contract VotingV2 is
         finder = FinderInterface(_finder);
         slashingLibrary = SlashingLibrary(_slashingLibrary);
         setSpamDeletionProposalBond(_spamDeletionProposalBond);
+
+        assembly {
+            sstore(priceRequestIds.slot, _startingRequestIndex)
+        }
```

Comment:
Similar to my comment above, we assume that indices in this array are < type(uint64).max. Should we ensure that we always have plenty of room between our starting index and type(uint64).max? Maybe we can require it's smaller than `type(uint64).max / 2` or something?